scipy/special/tests/test_logsumexp.py

```diff
@@ -19,7 +19,7 @@ def test_logsumexp():
     assert_almost_equal(logsumexp(b), desired)
 
     n = 1000
-    b = np.ones(n) * 10000
+    b = np.full(n, 10000, dtype='float64')
```

Comment:
Please use `np.float64`, the string-dtype specifiers are Numeric backward compat cruft...